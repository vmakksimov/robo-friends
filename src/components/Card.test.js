import { shallow } from "enzyme";
import Card from "./Card";
import React from "react";


it('testing shallow', () => {
    const mockRobots = [
        {
            id: 1,
            name: "John Snow",
            username: "JohnJohn",
            email: "John@gmail.com"
        }
    ]
    expect(shallow(<Card robots={mockRobots}/>)).toMatchSnapshot();
})

