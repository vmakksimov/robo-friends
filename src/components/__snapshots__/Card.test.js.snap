// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing shallow 1`] = `ShallowWrapper {}`;
